packages/lodestar/test/unit/network/gossip/scoringParameters.ts

```diff
@@ -0,0 +1,254 @@
+import {ATTESTATION_SUBNET_COUNT, ForkName, SLOTS_PER_EPOCH} from "@chainsafe/lodestar-params";
+import sinon from "sinon";
+import {SinonStubbedInstance} from "sinon";
+import {createIChainForkConfig} from "@chainsafe/lodestar-config";
+import {mainnetChainConfig} from "@chainsafe/lodestar-config/presets";
+import {ForkDigestContext} from "../../../../src/util/forkDigestContext";
+import {GossipPeerScoreParamsBuilder, gossipScoreThresholds} from "../../../../src/network/gossip/scoringParameters";
+import {WinstonLogger} from "@chainsafe/lodestar-utils";
+import {expect} from "chai";
+import {stringifyGossipTopic} from "../../../../src/network/gossip/topic";
+import {GossipType} from "../../../../src/network";
+import {TopicScoreParams} from "libp2p-gossipsub/src/score";
+
+/**
+ * Refer to Teku tests at
+ * https://github.com/ConsenSys/teku/blob/e18ab9903442410aa04b590c4cc46734e13d3ffd/networking/eth2/src/test/java/tech/pegasys/teku/networking/eth2/gossip/config/GossipScoringConfiguratorTest.java#L38
+ */
+export function runGossipScoringParametersTests(): void {
```

Comment:
this test needs to run with mainnet profile so I have to call it through `mainnetTest.ts`, any better ways to do it?

Reply:
Yeah, can you place the tests under `test/unit-mainnet` for consistency with other tests? Then add to package json
```
"test:unit": "LODESTAR_PRESET=mainnet mocha 'test/unit/**/*.test.ts'"
```